avocado/utils/memory.py

```diff
@@ -278,3 +279,35 @@ def get_buddy_info(chunk_sizes, nodes="all", zones="all"):
             buddyinfo_dict[chunk_size] += int(chunk_info)
 
     return buddyinfo_dict
+
+
+def set_thp_value(value, feature):
+    """
+    Sets THP feature to a given value
+
+    :param value : Value to be set to feature
+    :type value : str
+    :param feature : Thp feature to set
+    :type feature : str
+    """
+    thp_path = '/sys/kernel/mm/transparent_hugepage/'
+    thp_feature_to_set = os.path.join(thp_path, feature)
+    cmd = 'echo %s > %s ' % (value, thp_feature_to_set)
```

Comment:
Btw feel free to add `safe_write_file` which reports `AssertError` (or `RuntimeError`) to the `genio` library and use it here. I see the need for it on many places...